src/pow.cpp

```diff
@@ -428,6 +432,8 @@ RCP<const Basic> pow_expand(const RCP<const Pow> &self)
         }
     }
     RCP<const Basic> result = Add::from_dict(add_overall_coeff, std::move(rd));
+    RCP<const Number> i = integer(-1);
+    if(negative_pow) result = rcp(new Pow(result, i));
```

Comment:
I think this could cause some assert errors. Consider a case like 1/x^2, so `result = x^2` and then you would get `(x^2)^(-1)`, which will trigger an assert error, as it should rather be `x^(-2)`. You can achieve that by calling `pow(result, i)`.